classes/core/Core.inc.php

```diff
@@ -42,6 +42,24 @@ static function getBaseDir() {
 		return $baseDir;
 	}
 
+	/**
+	 * Get the path in which file caches will be stored.
+	 * @return string The full path to the file cache directory
+	 */
+	static function getFileCachePath() {
+		return Core::getBaseDir() . DIRECTORY_SEPARATOR . 'cache';
+	}
+
+	/**
+	 * Get a stash pool instance to use for file-based caching.
+	 * @return Stash\Driver\FileSystem A stash pool instance.
+	 */
+	static function getStashDriver() {
+		return new Stash\Driver\FileSystem(array(
+			'path' => Core::getFileCachePath() . DIRECTORY_SEPARATOR . 'stash',
+		));
+	}
```

Comment:
So a plugin that wanted to leverage caching could just do:

```php
$pool = new Stash\Pool(new Stash\Driver\FileSystem([
  'path' => Core::getFileCachePath() . DIRECTORY_SEPARATOR . 'myPluginName',
]);
```

Any benefit to making a `getStashDriver()` method on `Plugin` that enforced use of `<genre>/<pluginName>`?

Reply:
No, I think that should be done on a cache ID level (we can construct these arbitrarily) rather than in a different storage path. The caches should all live in the same Stash umbrella so that the same administrator's "clear cache" button can manage them all.

Reply:
Ah good point.